mathlibtools/leanproject.py

```diff
@@ -303,19 +318,41 @@ def import_graph(to: Optional[str], from_: Optional[str], output: str) -> None:
     By default the graph will be written to 'import_graph.dot'.
     For .dot, .pdf, .svg, or .png output you will need to install 'graphviz' first.
     """
-    project = proj()
-    graph = project.import_graph
-    if to and from_:
-        G = graph.path(start=from_, end=to)
-    elif to:
-        G = graph.ancestors(to)
-    elif from_:
-        G = graph.descendants(from_)
-    else:
-        G = graph
+    G = _import_graph_(to, from_)
     G.write(Path(output))
 
 
+@cli.command()
+@click.option('--sed', 'sed', default=False, is_flag=True,
+              help='Instead of printing a list of removable imports, print a sed script that can be run to remove the imports.')
```

Comment:
Emitting a git patch file might make the most sense, as almost all users of lean will have git installed.

Reply:
Agreed, it's probably a little more work though to determine the file offsets? Maybe the best thing to do is make the replacement using python itself

Reply:
Actually probably grep can do what sed does here even